mock/cpp_mock_scenarios/src/metrics/traveled_distance.cpp

```diff
@@ -30,10 +29,7 @@ class TraveledDistanceScenario : public cpp_mock_scenarios::CppScenarioNode
 {
 public:
   explicit TraveledDistanceScenario(const rclcpp::NodeOptions & option)
-  : cpp_mock_scenarios::CppScenarioNode(
-      "traveled_distance",
-      ament_index_cpp::get_package_share_directory("cargo_delivery") + "/maps/kashiwa",
```

Comment:
.osm file should be exactly fixed in scenario.
So, please specify it in constructor.